server/webapp/localize.xml

```diff
@@ -157,6 +157,9 @@
     <entry key="ENV_ADD_FAILED">Failed to add environment. {0}</entry>
     <entry key="ENV_UPDATE_FAILED">Failed to update environment ''{0}''. {1}</entry>
     <entry key="ENV_DELETE_FAILED">Failed to delete environment ''{0}''. {1}</entry>
+    <entry key="CONFIG_REPO_ADD_FAILED">Failed to add config repository. {0}</entry>
```

Comment:
I think there is `RESOURCE_DELETE_SUCCESSFUL` , `RESOURCE_NOT_FOUND`, `RESOURCE_ALREADY_EXISTS`. Can you create the required `RESOURCE...` and use that instead of messages for individual entities? I see that this was done for environment as well. You can refactor both to have generic messages.